### DISCUSSION TITLE
Getting responce without quoting

### DISCUSSION THREAD

**Discussion by igupalo** [n3d1117/chatgpt-telegram-bot#102] [Q&A]
Hi, how can i get responses from the bot but without quoting my message? just a regular responce.
thanks!

**Comment by BjornB2** [comment]
I came here to ask just that. Would be nice for 1 on 1 conversations.

**Comment by n3d1117** [comment]
Hey there @igupalo and @BjornB2, it should be enough to replace every occurrence of:
```
reply_to_message_id=update.message.message_id
```
with:
```
reply_to_message_id=None
```

In `telegram_bot.py` file. Let me know if it works for you and I can add it as a configuration option!

**Reply by BjornB2** [reply]
Thanks for the suggestion. I'm running the script using the docker container (https://registry.hub.docker.com/r/n3d1117/chatgpt-telegram-bot/) on a Synology NAS. I seem to be unable to open a terminal from within the gui and tbh I have no clue where the docker is actually saved on the Synology. @igupalo can you maybe try this?

**Comment by BjornB2** [comment]
Would it be possible to add this in the next version? Quoting looks a bit silly in a 1 on 1 chat 😊

**Reply by n3d1117** [reply]
Sure! Just to clarify, this only applies to private chats right? Group chat should always include quoting?

**Reply by BjornB2** [reply]
Yes! In groups it makes sense to quote, in private chats  not so much.

**Reply by n3d1117** [reply]
Done! You can now set `ENABLE_QUOTING="false"` in your .env file to disable quoting in private chats! (it's true by default)

**Reply by BjornB2** [reply]
I don't see a new release. Is this a hidden feature in the .2.2 release already?

**Reply by n3d1117** [reply]
> I don't see a new release. Is this a hidden feature in the .2.2 release already?

No, it's on the main branch for now. New release in a few days!

**Comment by BjornB2** [comment]
Ah ok! Thanks again!

**Comment by BjornB2** [comment]
Works like a charm! Thanks again!
